### ISSUE TITLE
[FEATURE] Support curve fi pools for token price

### ISSUE THREAD

**Issue by v1nvn** [credmark/credmark-models-py#90]
**Is your feature request related to a problem? Please describe.**
I am not able to get token price for tokens on curve fi pools

**Describe the solution you'd like**
Implement `curve.get-pool-price-info` model and aggregate it along with uniswap v2/v3 and sushi pools in `token.pool-price-info` model

**Comment by leafyoung** [comment]
Yup. I have partly addressed this in https://github.com/credmark/credmark-models-py/blob/ef5941112b8bad844a929ad6659fb0059b58a4da/models/credmark/protocols/dexes/curve/curve_finance.py#L263

This will be in my models' pipeline.
